dbtmetabase/__init__.py

```diff
@@ -6,112 +6,165 @@
         include_tags=include_tags,
-        docs_url=dbt_docs_url,
+        dbt_docs_url=dbt_docs_url,
+    )
+
+    # Sync and attempt schema alignment prior to execution; if timeout is not explicitly set, proceed regardless of success
+    if not metabase_config.metabase_sync_skip:
+        if metabase_config.metabase_sync_timeout is not None and not mbc.sync_and_wait(
+            metabase_config.metabase_database,
+            dbt_models,
+            metabase_config.metabase_sync_timeout,
+        ):
+            logging.critical("Sync timeout reached, models still not compatible")
+            return
+
+    # Process Metabase stuff
+    mbc.export_models(
+        database=metabase_config.metabase_database,
+        models=dbt_models,
+        aliases=reader.catch_aliases,
+    )
+
+
+def exposures(
+    metabase_config: MetabaseConfig,
+    dbt_config: dbtConfig,
+    output_path: str,
+    output_name: str,
+    include_personal_collections: bool = False,
+    exclude_collections: Optional[Iterable] = None,
+):
+    """Extracts and imports exposures from Metabase to dbt.
+
+    Args:
+        metabase_config (str): Source database name.
+        dbt_config (str): Target Metabase database name. Database in Metabase is aliased.
+        output_path (str): Append the dbt tags to the end of the table description. Defaults to True.
+        output_name (str): URL to your dbt docs hosted catalog, a link will be appended to the model description (only works for manifest parsing). Defaults to None.
+        include_personal_collections (bool, optional): Model names to limit processing to. Defaults to None.
+        exclude_collections (Iterable, optional): Model names to exclude. Defaults to None.
+    """
+
+    # Assertions
+    if dbt_config.dbt_path and dbt_config.dbt_manifest_path:
+        logging.warning("Prioritizing manifest path arg")
+        dbt_config.dbt_path = None
+    if dbt_config.dbt_path and not dbt_config.schema:
+        logging.error(
+            "Must supply a schema if using YAML parser, it is used to resolve foreign key relations and which Metabase models to propagate documentation to"
+        )
+
+    # Instantiate Metabase client
+    mbc = MetabaseClient(
+        host=metabase_config.metabase_host,
+        user=metabase_config.metabase_user,
+        password=metabase_config.metabase_password,
+        use_http=metabase_config.metabase_use_http,
+        verify=metabase_config.metabase_verify,
+    )
+    reader: Union[DbtFolderReader, DbtManifestReader]
+
+    # Resolve dbt reader being either YAML or manifest.json based
+    if dbt_config.dbt_manifest_path:
+        reader = DbtManifestReader(os.path.expandvars(dbt_config.dbt_manifest_path))
+    elif dbt_config.dbt_path:
+        reader = DbtFolderReader(os.path.expandvars(dbt_config.dbt_path))
+
+    if dbt_config.schema_excludes:
+        dbt_config.schema_excludes = {
+            schema.upper() for schema in dbt_config.schema_excludes
+        }
+
```

Comment:
To troubleshoot I added code here to print out dbt_config.schema and dbt_config.schema_excludes. 

With this invocation:
dbt-metabase exposures \
--dbt_manifest_path dbt_manifest_semantic_types/manifest.json \
--dbt_database ANALYTICS \
--metabase_host localhost:3000 \
--metabase_user eep@... \
--metabase_password '...' \
--metabase_database EPullum_Dev \
--metabase_use_http \
--schema DEV_EPULLUM 

dbt_config.schema has a value of None
dbt_config.schema_excludes has a value of {'DEV_EPULLUM'}

Reply:
@erika-e 

Thanks for your diligence, you rock.
Your totally right. I am managing two PRs right now and I during my own code review processes would've caught and fixed the issue but only applied the change to my latest branch `metrics` 

I will add a commit today that addresses.

Reply:
Happy to help! 😃   Exposures parsing is going to be an awesome tool for us so I'm glad to be able to support and contribute.